src/playback/internal/playbackuiactions.cpp

```diff
@@ -72,7 +72,7 @@ const UiActionList PlaybackUiActions::m_settingsActions = {
     UiAction("midi-on",
              mu::context::UiCtxAny,
              mu::context::CTX_ANY,
-             TranslatableString("action", "MIDI input"),
+             TranslatableString("action", "Enable MIDI input"),
              TranslatableString("action", "Toggle ‘Enable MIDI input’"),
```

Comment:
@bkunda How about changing `Toggle ‘Enable MIDI input’` to just `Toggle MIDI input`?

Reply:
I think just go with `Enable MIDI input` (the check mark alongside it is sufficient to indicate whether this is enabled or not).

Reply:
@bkunda Yes, for in the menu `Enable MIDI input` is good, together with the checkmark. 

But I was wondering about Preferences, because there is no checkmark of course. So I could imagine using "Toggle" there, to emphasize that it is an on/off thing. On the other hand, it might be confusing to use a different name in Preferences than in the menu, maybe in this particular case even slightly more than in other cases. So what do you think?